buildenvs/base.Dockerfile

```diff
@@ -0,0 +1,143 @@
+# COPY --from=gcc-arm /include/                                        /include
+# COPY --from=gcc-arm /lib/gcc/                                        /lib/gcc
+# COPY --from=gcc-arm /arm-linux-gnueabihf                             /arm-linux-gnueabihf
+# COPY --from=gcc-arm /libexec/gcc/arm-linux-gnueabihf/12.2.0/cc1      /libexec/gcc/arm-linux-gnueabihf/12.2.0/cc1
+# COPY --from=gcc-arm /libexec/gcc/arm-linux-gnueabihf/12.2.0/collect2 /libexec/gcc/arm-linux-gnueabihf/12.2.0/collect2
+# COPY the others if needed
+
+# COPY --from=gcc-arm64 /bin/                                          /bin
+# COPY --from=gcc-arm64 /lib/gcc/                                      /lib/gcc
+# COPY --from=gcc-arm64 /include/                                      /include
+# COPY --from=gcc-arm64 /lib/gcc/                                      /lib/gcc
+# COPY --from=gcc-arm64 /aarch64-linux-gnu/                            /aarch64-linux-gnu
+# COPY --from=gcc-arm64 /libexec/gcc/aarch64-linux-gnu/12.2.0/cc1      /libexec/gcc/aarch64-linux-gnu/12.2.0/cc1
+# COPY --from=gcc-arm64 /libexec/gcc/aarch64-linux-gnu/12.2.0/collect2 /libexec/gcc/aarch64-linux-gnu/12.2.0/collect2
+# COPY the others if needed
+
+# Link the GCC toolchain
+RUN set -xe; \
+    ln -s /bin/${GCC_PREFIX}-as         /bin/as; \
+    ln -s /bin/${GCC_PREFIX}-ar         /bin/ar; \
+    ln -s /bin/${GCC_PREFIX}-c++        /bin/c++; \
+    ln -s /bin/${GCC_PREFIX}-c++filt    /bin/c++filt; \
+    ln -s /bin/${GCC_PREFIX}-elfedit    /bin/elfedit; \
+    ln -s /bin/${GCC_PREFIX}-gcc        /bin/cc; \
+    ln -s /bin/${GCC_PREFIX}-gcc        /bin/gcc; \
+    ln -s /bin/${GCC_PREFIX}-gcc-ar     /bin/gcc-ar; \
+    ln -s /bin/${GCC_PREFIX}-gcc-nm     /bin/gcc-nm; \
+    ln -s /bin/${GCC_PREFIX}-gcc-ranlib /bin/gcc-ranlib; \
+    ln -s /bin/${GCC_PREFIX}-gccgo      /bin/gccgo; \
+    ln -s /bin/${GCC_PREFIX}-gcov       /bin/gcov; \
+    ln -s /bin/${GCC_PREFIX}-gcov-dump  /bin/gcov-dump; \
+    ln -s /bin/${GCC_PREFIX}-gcov-tool  /bin/gcov-tool; \
+    ln -s /bin/${GCC_PREFIX}-gprof      /bin/gprof; \
+    ln -s /bin/${GCC_PREFIX}-ld         /bin/ld; \
+    ln -s /bin/${GCC_PREFIX}-nm         /bin/nm; \
+    ln -s /bin/${GCC_PREFIX}-objcopy    /bin/objcopy; \
+    ln -s /bin/${GCC_PREFIX}-objdump    /bin/objdump; \
+    ln -s /bin/${GCC_PREFIX}-ranlib     /bin/ranlib; \
+    ln -s /bin/${GCC_PREFIX}-readelf    /bin/readelf; \
+    ln -s /bin/${GCC_PREFIX}-size       /bin/size; \
+    ln -s /bin/${GCC_PREFIX}-strings    /bin/strings; \
+    ln -s /bin/${GCC_PREFIX}-strip      /bin/strip;
+
+# Copy the QEMU emulator
+COPY --from=qemu /bin/                  /bin
+COPY --from=qemu /share/qemu/           /share/qemu
+COPY --from=qemu /lib/x86_64-linux-gnu/ /lib/x86_64-linux-gnu
+
+# COPY --from=qemu /lib/arm-linux-gnueabihf/ /lib/arm-linux-gnueabihf
+# COPY --from=qemu /lib/aarch64-linux-gnu/ /lib/aarch64-linux-gnu
+
+# Unikraft workdir and environment
+WORKDIR /usr/src/unikraft/apps/app
```

Comment:
This is an artefact from `pykraft`, we can assume the default debian working directory:
```diff
- # Unikraft workdir and environment
- WORKDIR /usr/src/unikraft/apps/app
```